src/Components/Home/ArticleList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "../ArticleCard";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-app-ftk2.onrender.com/api/articles")
+      .then((res) => {
+        setArticles(res.data.articles[0]);
+      });
+  }, []);
+
+  return (
+    <div className="w-8/12 mx-auto">
+      <ul className="flex flex-col gap-5">
+        {articles && articles.length > 0 ? (
```

Comment:
In order for articles to have a length of 0, it would have to exist in the first place. Therefore I think it's only necessary to have the first half of this assertion.